### PR TITLE
Perf system

### PR DESCRIPTION
**Pull request by jeffa5**

### CODE REVIEW

File: benchmark/common.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class Config:
     sig_ms_interval: int
     ledger_chunk_bytes: str
     snapshot_tx_interval: int
+    http: int
```
Comment (jumaffre) [inline benchmark/common.py:44]:
```suggestion
    http_version: int
```

**Review by jumaffre — APPROVED** [review]

**Comment by heidihoward** [comment]
Some graphs seem to be missing data?

**Comment by jeffa5** [comment]
> Some graphs seem to be missing data?

That will be because we use a default value for a few of them for etcd since most of the store parameters don't impact etcd. This means some of the plots don't have data in all combinations.

I think something for another PR

**Review by heidihoward — APPROVED** [review]

**Comment by jeffa5** [comment]
>  File name too long: 

Damn.

I guess I'll rework that then

**Pull request merged**